src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -210,36 +224,29 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
               />
             </FormControl>
           </Grid>
-          <Grid item container xs={6} spacing={2}>
-            <Grid item xs={12}>
-              <FormControl fullWidth>
-                <FormLabel>Expense Type</FormLabel>
-                <Controller
-                  name="expenseTypeId"
-                  control={control}
-                  render={({ field: { onChange, value } }) => {
-                    const mappedExpenseTypes = allExpenseTypes.map(expenseTypesToAutocomplete);
-                    return (
-                      <NERAutocomplete
-                        id={'expenseType'}
-                        size="medium"
-                        options={mappedExpenseTypes}
-                        value={mappedExpenseTypes.find((expenseType) => expenseType.id === value) || null}
-                        placeholder="Expense Type"
-                        onChange={(_event, newValue) => {
-                          newValue ? onChange(newValue.id) : onChange('');
-                        }}
-                      />
-                    );
-                  }}
-                />
-                <FormHelperText error>{errors.expenseTypeId?.message}</FormHelperText>
-              </FormControl>
-            </Grid>
-            <Grid item xs={12}>
-              <FormLabel>Total Cost</FormLabel>
-              <Typography variant="h6">${calculatedTotalCost}</Typography>
-            </Grid>
+          <Grid item xs={6}>
+            <FormControl fullWidth>
+              <FormLabel>Refund Source</FormLabel>
+              <Controller
+                name="account"
```

Comment:
U changed the name in the other file

Reply:
you mean the one I changed to allowed refund source? We just need that title in the settings, the user is still setting a refund source here, not an allowed refund source

Reply:
no i mean the name in the other autocomplete is expenseTypeId and this one is account